crates/network-scanner/Cargo.toml

```diff
@@ -8,12 +8,20 @@ publish = false
 
 [dependencies]
 anyhow = "1.0.79"
+crossbeam = { version = "0.8.4", features = ["crossbeam-channel"] }
+dashmap = "5.5.3"
```

Comment:
issue: Now unused

Reply:
this is another crates, where it used to store scanned IPs, it is unused in network-scan-net

Reply:
We probably don’t need dashmap in `network-scanner` either

Reply:
we might? we actually have a lot concurrent request

Reply:
I’ll be honest here, dashmap is probably not worth anything unless we hit at least 10 000 000 operations per seconds, and I highly doubt we’ll ever because we are I/O-bound
This kind of data structure is a useful building block for CPU-bound algorithms

Reply:
udpated